src/odb/include/odb/db.h

```diff
@@ -1689,6 +1689,12 @@ class dbBlock : public dbObject
 
  private:
   void ComputeBBox();
+
+  ///
+  /// Ensure that the constraint region defined in the given edge is completely
+  /// inside the die area.
+  ///
+  void ensureConstraintRegion(const Direction2D& edge, int& begin, int& end);
```

Comment:
I should have been more explicit.  I mean it should be a method of _dbBlock and not in db.h.